### DISCUSSION TITLE
Filter MPPS forwarding rule

### DISCUSSION THREAD

**Discussion by albertosaez** [dcm4che/dcm4chee-arc-light#2943] [Q&A] [answered]
Is it possible to configure the system to forward only some received MPPS notifications ? (based on the source aetitle  ? 

With the standard [config  for forwarding](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Forwarding-of-MPPS-N%E2%80%92CREATE-and-N%E2%80%92SET-messages) all received MPPS are forwarded, but we're interested to forward only a few of them based on the source AET.

**Comment by gunterze** [comment, marked as answer]
You may configure distinguish Archive AEs and configure MPPS forwarding only for particular Archive AEs.
